README.md

```diff
@@ -425,7 +428,7 @@ following properties:
       For example, `'No camera was found.'`
     
     * `qualityLimitationReason` (String) --  Indicates the reason behind
-       the highest resolution tested failing. It can have values: `'cpu'` for CPU overload, `'bandwidth'` for insufficient network bandwidth, or value is `'null'` if there is no limitation.
+       the highest resolution tested failing. It can have values: `'cpu'` for CPU overload, `'bandwidth'` for insufficient network bandwidth, or value is `null` if there is no limitation.
```

Comment:
This is set to `null` in Safari. But it is set to `'none'` in Chrome.

Reply:
@jeffswartz Fixed